diff -urN ./libcli-1.9.4/libcli.c ./libcli-1.9.4.new/libcli.c
--- ./libcli-1.9.4/libcli.c	2008-10-09 11:38:01.000000000 +0900
+++ ./libcli-1.9.4.new/libcli.c	2020-01-17 14:30:34.076779732 +0900
@@ -528,7 +528,7 @@
     if (!(cli = calloc(sizeof(struct cli_def), 1)))
         return 0;
 
-    cli->buf_size = 1024;
+    cli->buf_size = (1024 * 1024);
     if (!(cli->buffer = calloc(cli->buf_size, 1)))
     {
         free_z(cli);
